packages/analytics-core/src/plugins/destination.ts

```diff
@@ -95,65 +96,46 @@ export class Destination implements DestinationPlugin {
     });
   }
 
-  scheduleTryable(list: Context[], shouldAddToQueue = false) {
-    list.forEach((context) => {
-      // Only need to concat the queue for the first time
-      if (shouldAddToQueue) {
-        this.queue = this.queue.concat(context);
-      }
-      if (context.timeout === 0) {
-        this.schedule(this.config.flushIntervalMillis);
-        return;
-      }
-
-      setTimeout(() => {
-        context.timeout = 0;
-        this.schedule(0);
-      }, context.timeout);
-    });
-  }
-
-  addToQueue(...list: Context[]) {
-    const tryable = this.getTryableList(list);
-    this.scheduleTryable(tryable, true);
-    this.saveEvents();
-  }
-
-  schedule(timeout: number) {
+  sendEventIfReady() {
     if (this.scheduled || this.config.offline) {
       return;
     }
+    // if batching enabled, check if min threshold met for batch size
```

Comment:
when is batching not enabled and don't check min threshold? We can delete this comment if it's always enabled to ease confusion.

Reply:
Removed.